Formula/v/veryfasttree.rb

```diff
@@ -0,0 +1,76 @@
+class Veryfasttree < Formula
+  desc "Efficient phylogenetic tree inference for massive taxonomic datasets"
+  homepage "https://github.com/citiususc/veryfasttree"
+  url "https://github.com/citiususc/veryfasttree/archive/refs/tags/v4.0.3.tar.gz"
+  sha256 "770e41773806c65e60ebc77363beccf40e3d339affd97c07eb7b2e1acb51af60"
+  license "GPL-3.0-only"
+  head "https://github.com/citiususc/veryfasttree.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "libomp"
+
+  def install
+    system "cmake", "-B", "build", "-DUSE_NATIVE=OFF"
```

Comment:
What does `-DUSE_NATIVE=OFF` do?

Reply:
"-DUSE_NATIVE=OFF" is used to avoid compiling the binary with hardware-specific features of the computer where it's being compiled because, in this case, the Bottles are not generated on the same computer where they will run. However, if a user compiles the formula on their computer, it would be good to set it to "ON." It is possible to detect when a binary is being generated for a Bottle or when a user is compiling the formula locally for their computer. If possible, I would prefer to add an "if" statement in the recipe for both cases.

Reply:
Since compiling from source is unsupported that doesn't seem like a good idea, but since Homebrew shims ensure that the lowest supported hardware is used as a build target it seems like this should be set to `ON`

Reply:
Ok, removed, default is ON